container/client.go

```diff
@@ -0,0 +1,211 @@
+// +build linux
+
+/*
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2015 Intel Corporation
```

Comment:
I think it was agreed before dates in header files don't have to be updated while introducing changes to already released plugin.